corehq/apps/case_importer/do_import.py

```diff
@@ -56,7 +56,7 @@ def __init__(self, domain, config, task, record_form_callback):
 
     def do_import(self, spreadsheet):
         for row_num, row in enumerate(spreadsheet.iter_row_dicts(), start=1):
-            set_task_progress(self.task, row_num, spreadsheet.max_row)
+            set_task_progress(self.task, row_num - 1, spreadsheet.max_row)
```

Comment:
Even spookier - this bug only triggers the race condition when my branch is merged with master.  :man_shrugging: